src/Aguacongas.Identity.Firestore/README.md

```diff
@@ -0,0 +1,39 @@
+# Aguacongas.Identity.Firestore
+Google Firestore-based ASP.NET Core Identity user & role stores.
+
+## Basic usage
+1. Set `GOOGLE_APPLICATION_CREDENTIALS` environment variable to a path to the JSON file for GCP service account (see https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable)
+1. In `Startup.cs` of your ASP.NET Core project:
```

Comment:
duplicate 1.

Reply:
Markdown automatically numerates the lists:
https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
Maybe, but it's ugly. Don't you think ?

Reply:
I usually don't read raw markdowns :D 
Changed as per request